apps/preview/next/pages/showcases/Banners/DisplayHorizontal.tsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable jsx-a11y/anchor-has-content */
+import { ShowcasePageLayout } from '../../showcases';
+// #region source
+import { SfButton } from '@storefront-ui/react';
+import classNames from 'classnames';
+import sunglasses from '@assets/display.png';
+import backpack from '@assets/display-2.png';
+import winterHat from '@assets/display-3.png';
+
+const displayDetails = [
+  {
+    image: sunglasses,
+    title: 'Sunny Days Ahead',
+    subtitle: 'Be inspired',
+    description: 'Step out in style with our sunglasses collection',
+    buttonText: 'Discover now',
+    reverse: false,
+    backgroundColor: 'bg-negative-200',
+    titleClass: 'md:typography-headline-2',
+    subtitleClass: 'md:typography-headline-6',
+    descriptionClass: 'md:typography-text-lg',
+  },
+  {
+    image: backpack,
+    title: 'Pack it Up',
+    subtitle: 'Be active',
+    description: 'Explore the great outdoors with our backpacks',
+    buttonText: 'Discover now',
+    reverse: true,
+    backgroundColor: 'bg-warning-200',
+  },
+  {
+    image: winterHat,
+    title: 'Fresh and Bold',
+    subtitle: 'New collection',
+    description: 'Add a pop up color to your outfit',
+    buttonText: 'Discover now',
+    reverse: false,
+    backgroundColor: 'bg-secondary-200',
+  },
+];
+
+export default function DisplayHorizontalBlock() {
+  return (
+    <div className="flex flex-col md:flex-row flex-wrap gap-6 max-w-[1540px]">
+      {displayDetails.map(
+        ({ image, title, subtitle, description, buttonText, backgroundColor, reverse, titleClass, subtitleClass }) => (
+          <div
+            key={title}
+            className={classNames(
+              'relative flex md:max-w-[1536px] md:[&:not(:first-of-type)]:flex-1 md:first-of-type:w-full',
+              backgroundColor,
+            )}
+          >
+            <a
+              className="absolute w-full h-full z-1 focus-visible:outline focus-visible:rounded-lg"
+              aria-label={title}
+              href="/"
+            />
+            <div
+              className={classNames('flex items-center justify-between overflow-hidden grow', {
```

Comment:
```suggestion
              className={classNames('flex justify-between overflow-hidden grow', {
```
when removing `items-center` nothing changes, is it needed?

Reply:
I'll check items-center and I think overflow-hidden is needed when display has less text lines

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2564/commits/5dcd99203fcebe2bf08f3261d030601c6f0fb320 removed items-center